README.md

```diff
@@ -1 +1,174 @@
-# Leaderboard
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+  
+  <img src="./src/Logo_HG_ReadmeFile.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+<h3><b>README</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Leaderboard\] ](#-leaderboard-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+# 📖 [Leaderboard] <a name="about-project"></a>
+
+This project is about using an API to get a list of names and their scores. 
+
+**[Leaderboard]** For this project, I used Gitflow by working on a feature branch and using develop as my main branch till production, and also use webpack as compiler.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Style</summary>
+  <ul>
+    <li><a href="#">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Dynamic</summary>
+  <ul>
+    <li><a href="#">JavaScript</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Standar code</summary>
+  <ul>
+    <li><a href="#">Linters</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Use GitFlow]**
+- **[Use Linters]**
+- **[Use ES6 syntax]**
+- **[Use JavaScript Modules]**
+- **[Use Webpack Template]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Open on Visual Studio Code or any other code reader, and use an app as Live Server to review online.
+### Setup
+
+Clone this repository to your desired folder:
+
+git clone https://github.com/HFG43/Leaderboard
+
+### Install
+
+Open index.html file with your Code Editor, and review the behaviour of the page under different sizes.
+
+### Usage
+
+To run the project, execute the following command:
+
```

Comment:
- [ ] I kindly suggest that you add the step for usage of your web app. for example
[OPTIONAL]
```
### Usage

To run the project, execute the following command:


- Start the development server
`   ```sh
`   npm start
`   ```
`
```